### PR TITLE
(2048) Show latest draft or live version on admin side

### PR DESCRIPTION
**Pull request by pezholio**
This changes the behaviour of the admin side to fetch the latest version of an organisation (either draft or live) on the listing page, and then links through to show the version, rather than the organisation. This should now have fixed all the failing e2e tests we had skipped, so once this is in, we should be in a position to merge the `organisation-versioning` branch to main 🙌

### CODE REVIEW

File: src/organisations/organisation-versions.service.ts [hunk 1]
```diff
@@ -51,6 +51,20 @@ export class OrganisationVersionsService {
     );
   }
 
+  async allDraftOrLive(): Promise<Organisation[]> {
```
Comment (bryangaledxw) [inline src/organisations/organisation-versions.service.ts:65]: Can this end up returning archived/unconfirmed versions?

File: src/organisations/presenters/organisation.presenter.ts [hunk 2]
```diff
@@ -27,6 +27,11 @@ export class OrganisationPresenter {
       {
         html: await this.industries(),
       },
+      {
+        html: await this.i18nService.translate(
```
Comment (bryangaledxw) [inline src/organisations/presenters/organisation.presenter.ts:31]: If an org has a live version and a later draft version, is the status definitely draft? I feel like a user would see "draft" to mean "not live"

File: src/organisations/organisation-versions.service.spec.ts [hunk 3]
```diff
@@ -253,3 +259,43 @@ describe('OrganisationVersionsService', () => {
     });
   });
 });
+
+expect.extend({
```
Comment (bryangaledxw) [inline src/organisations/organisation-versions.service.spec.ts:263]: Will this be useful outside this test? Is there a more global place this could live?

**Review by bryangaledxw — APPROVED** [review]
Looks good, but there's a couple of things I just want to check

File: src/organisations/organisation-versions.service.ts (hunk 1 above)
Comment (pezholio) [reply src/organisations/organisation-versions.service.ts:65]: Oh, good call. I missed that!

File: src/organisations/presenters/organisation.presenter.ts (hunk 2 above)
Comment (pezholio) [reply src/organisations/presenters/organisation.presenter.ts:31]: This is referring to the version that's visible in the admin interface, but I get where you're coming from. Let's see what comes out in testing 👍

File: src/organisations/organisation-versions.service.spec.ts (hunk 3 above)
Comment (pezholio) [reply src/organisations/organisation-versions.service.spec.ts]: Good call - I've added another commit for this :+1

**Pull request merged**